addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -170,5 +182,22 @@ weSnippetEditor.Class.include({
             tab: this.tabs.THEME,
         });
     },
+    /**
+     * Called when search input value changed > adapts the snippets grid.
+     */
+    _onSearchFilterInputChange: function () {
+        var self = this;
```

Comment:
You probably never will need to do `var self = this;` anywhere anymore. Use `this` directly when possible.